source/lovelace/index.markdown

```diff
@@ -31,3 +31,4 @@ To try it yourself, check out [the demo](https://demo.home-assistant.io).
 
 * [Awesome HA](https://www.awesome-ha.com/#lovelace-ui)
 * [Community Custom Cards](https://github.com/custom-cards)
+* [Material Design Icons](https://cdn.materialdesignicons.com/4.4.95/)
```

Comment:
Maybe we should avoid having the version in the URL? Since that means we need to keep it updated in the future.
How about linking it to just: <https://materialdesignicons.com/> ?

Reply:
That makes sense, how about showing the community by default since those are the ones we can use in HA.
[https://materialdesignicons.com/tag/community](https://materialdesignicons.com/tag/community)